src/middleware/node/index.ts

```diff
@@ -12,9 +12,15 @@ export function createNodeMiddleware(
     log = createLogger(),
   }: MiddlewareOptions = {}
 ) {
+  const deprecateOnUnhandledRequest = (request: any, response: any) => {
+    console.error(
```

Comment:
I know this has been merged, but shouldn't this be a warning instead of an error? It being an error seems to confuse Jest in some tests I run, and a warning would fit here better in my opinion.